cli/src/semgrep/error.py

```diff
@@ -171,10 +170,6 @@ def is_timeout(self) -> bool:
         """
         return isinstance(self.core.error_type.value, out.Timeout)
 
-    # TODO: remove this SemgrepCoreError prefix
-    def semgrep_error_type(self) -> str:
-        return f"SemgrepCoreError: {self._error_type_string()}"
```

Comment:
@zzeleznick @nmote I'm removing this SemgrepCoreError prefix here; you might need to update
some metabase queries.